sn_transfers/src/genesis.rs

```diff
@@ -145,18 +135,12 @@ pub fn get_genesis_sk() -> MainSecretKey {
     }
 }
 
-/// Return if provided Transaction is genesis parent tx.
-pub fn is_genesis_parent_tx(parent_tx: &Transaction) -> bool {
-    parent_tx == &*GENESIS_CASHNOTE_PARENT_TX
-}
-
 /// Return if provided Spend is genesis spend.
 pub fn is_genesis_spend(spend: &SignedSpend) -> bool {
     let bytes = spend.spend.to_bytes_for_signing();
     spend.spend.unique_pubkey == *GENESIS_SPEND_UNIQUE_KEY
         && GENESIS_SPEND_UNIQUE_KEY.verify(&spend.derived_key_sig, bytes)
-        && is_genesis_parent_tx(&spend.spend.parent_tx)
-        && spend.spend.amount == NanoTokens::from(GENESIS_CASHNOTE_AMOUNT)
+        && spend.spend.amount() == NanoTokens::from(GENESIS_CASHNOTE_AMOUNT)
 }
```

Comment:
If we know the content of the Genesis Spend, maybe also add those values in to make sure it's the only and unique genesis?
```
&& spend.spend.reason == GENESIS_REASON
&& spend.spend.ancestors == GENESIS_ANCESTORS
&& spend.spend.descendants == GENESIS_DESCENDANTS
```